frontend/packages/console-dynamic-plugin-sdk/src/api/api-types.ts

```diff
@@ -43,3 +43,15 @@ export type UseK8sWatchResources = <R extends ResourcesObject>(
 export type UseResolvedExtensions = <E extends Extension>(
   ...typeGuards: ExtensionTypeGuard<E>[]
 ) => [ResolvedExtension<E>[], boolean, any[]];
+
+export type CoFetch = (url: string, options?: RequestInit, timeout?: number) => Promise<Response>;
```

Comment:
@christianvogt @vojtechszocs Do you guys think we should keep the `co` prefix or should we consider a different name?

Reply:
Maybe `ConsoleFetch` instead of `CoFetch` ?

Reply:
Updated to `ConsoleFetch`.